api/notification.proto

```diff
@@ -414,4 +427,17 @@ service notification {
 	}
 
 
+	rpc CreateWebsocketMessage (CreateWebsocketMessageRequest) returns (CreateWebsocketMessageResponse) {
```

Comment:
Can this func merged into CreateNotification?

Reply:
yes